code/modules/mining/aux_base.dm

```diff
@@ -193,8 +197,8 @@ interface with the mining shuttle at the landing site if a mobile beacon is also
 	to_chat(user, "<span class='notice'>You begin setting the landing zone parameters...</span>")
 	setting = TRUE
 	if(!do_after(user, 50, target = user)) //You get a few seconds to cancel if you do not want to drop there.
-		setting = FALSE
 		return
+	setting = FALSE
```

Comment:
TL;DR: Accidental deletion.

Long story: It is to fix the remote becoming useless if it failed one of the three turf checks. The `setting` var is only meant to stop the user spam clicking the thing during the `do_after`. I used a dirty master that I merged with the PK borg PR from Kevinz. I did not notice until I went to PR it, so I had to roll back my changes, clean out the Kevinz code, and rebuild what I had done the first time. I did not restore the correct version of my pre-Kev-fix backup, resulting in a move instead of a copy.